src/scss/vendors/_vendors.scss

```diff
@@ -89,44 +106,14 @@
 	}
 }
 
-// To adapt to UIO high contrast themes
-[class^="gpii-ext-theme"],
-[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) {
-	// WeCount logo
-	.logo circle,
-	.logo path {
-		fill: none;
-		stroke: var(--logo-stroke, currentColor);
-		stroke-linecap: round;
-		stroke-width: rem(3);
-	}
-
-	// Search icon; IDRC, OCADU and Canada logos
-	header .site-nav-wrapper .site-nav .search-container svg {
-		fill: currentColor;
-	}
-
-	// Canada logo
-	--theme-link-fill: currentColor;
-
-	// Placeholder images on the "views" page
-	svg.news-item-img use {
-		stroke: var(--placeholder-stroke, currentColor);
-		stroke-linecap: round;
-		stroke-width: rem(3);
-	}
-}
-
-// For styles that require foreground and background colors from each theme
-@each $theme, $fColor, $bColor in $UIOThemes {
-	.fl-theme-#{$theme}, .gpii-ext-theme-#{$theme} {
-		// WeCount logo
-		--logo-stroke: #{$fColor};
-
-		// The placeholder svg on the "views" page
-		--theme-background-fill: #{$bColor};
-		--placeholder-stroke: #{$fColor};
-
-		@include adapt-to-highContrast($fColor, $bColor);
+@media screen and (min-width: 1024px) {
+	[class^="gpii-ext-theme"],
+	[class^="fl-theme"]:not(.fl-theme-prefsEditor-default) {
+		// Search icon; IDRC, OCADU and Canada logos
+		header .site-nav-wrapper .site-nav .search-container svg {
```

Comment:
I believe this css needs to be moved out of the generic selector into theme specific selectors too. Seems you need another loop on `$UIOThemes` here.